utilMenu.java

```diff
@@ -40,6 +40,10 @@ public static void stareaContului(String ID) {
 
     //metoda care ajuta metoda-mama optiuni
     public static void modificareOptiune(String ID, int pret, String Optiunea, String MINGB, int cantitatea) {
+        int x;
+        if (Objects.equals(MINGB, "MIN")) x = 2;
+        else x = 1;
+
```

Comment:
Logica la ce ai facut aici este urmatoarea:
creezi o variabila x, daca MINGB este egal cu "MIN" atunci x va fi 2, daca nu 1

in cazul tau concret, poti sa creezi x sa fie 1 by default, daca MINGB == "MIN" atunci valoarea se schimba in 2, de altfel ramane 1.


Asta e parte de un concept de optimizare a codului numit "Reversing conditions"
 poti sa citesti un pic cand ai timp, destul de util, dar deja este o chestie de stilizare a codului, nu e obligatoriu, dar ar fi bine sa stii